snappy_pipeline/workflows/gene_expression_quantification/__init__.py

```diff
@@ -211,7 +211,7 @@ def args_function(wildcards):
         assert action == "run", "Unsupported actions"
         return args_function
 
-    def _collect_reads(self, wildcards, library_name, prefix):
+    def _collect_reads(self, wildcards, _library_name, prefix):
```

Comment:
Rather use the following to suppress errors. Prefixing arguments with underscores for sub classes blows up when people start using kwargs style argument passing.

```python
_ = library_name
```